js/web/lib/wasm/jsep/webgpu/ops/3rd-party/matmul_packed_webgpu.ts

```diff
@@ -510,11 +510,7 @@ export const createMatmulProgramInfo =
         name: 'MatMul',
         shaderCache: {
           hint: activationAttributes.activationCacheKey + `${elementsPerThread}` +
-              `${activationAttributes.activation}` +
-              `${activationAttributes.clipMax}` +
-              `${activationAttributes.clipMin}` +
```

Comment:
do you remove activation attributes from cache hint intentionally?

Reply:
Yes, `activationAttributes.activationCacheKey` has included those information. It's not necessary to duplicate them.